js/src/features/project/components/ProjectHeaderMenu.tsx

```diff
@@ -1,37 +1,72 @@
 import React from "react";
-import { Menu } from "antd";
+import { Menu, PageHeader, Tag, Typography } from "antd";
 import { Link, useLocation } from "react-router-dom";
+import { useSelector } from "react-redux";
+
+import { selectProject } from "../selectors";
+import { ProjectStatusTypeResponse } from "ProjectModels";
+import { chooseColor } from "../helpers";
+
+import "../project.scss";
+
+const { Paragraph } = Typography;
 
 type Props = {
   slug: string;
 };
 
 export const ProjectHeaderMenu: React.FC<Props> = ({ slug }) => {
   const location = useLocation();
+  const project = useSelector(selectProject);
 
   return (
-    <Menu
-      mode="horizontal"
-      selectedKeys={[location.pathname]}
-      className="project-menu"
+    <PageHeader
+      className="page-header"
+      title={project?.name}
+      tags={<Status status={project?.status} />}
+      footer={
+        <Menu
+          mode="horizontal"
+          selectedKeys={[location.pathname]}
+          className="page-header__menu"
+        >
+          <Menu.Item key={`/projects/${slug}/role`}>
+            <Link to={`/projects/${slug}/role`}>
+              <span>Role</span>
+            </Link>
+          </Menu.Item>
+          <Menu.Item
+            key={location.pathname.includes("teams") ? location.pathname : ""}
```

Comment:
Why do you check only `teams` url ?

Reply:
If I am on the page of a specific project, then I need the tab to be highlighted "Teams". You cannot put an array in `key` for verification, so I did such a verification.

Reply:
But what about roles or settings?

Reply:
I checking roles and setting
`key={'/projects/${project.slug}/role'}`
`key={'/projects/${project.slug}/settings'}`